incubator/buzzfeed-sso/templates/ingress.yaml

```diff
@@ -40,10 +40,12 @@ spec:
     - host: {{ .domain }}
       http:
         paths:
-          - path: {{ .path }}
+          {{- range .paths }}
+          - path: {{ . }}
             backend:
               serviceName: {{ $fullName }}-proxy
               servicePort: http
+          {{- end }}
```

Comment:
Good idea. Let's make this backwards compatible with the current clients and then will be ready to go